lib/orders_table.sql

```diff
@@ -0,0 +1,20 @@
+
+-- file: order_table.sql
+
+
+CREATE TABLE orders (
+  id SERIAL PRIMARY KEY,
+  customer TEXT,
+  date DATE
```

Comment:
Looks like you looked up https://www.postgresql.org/docs/current/datatype-datetime.html so you either already knew the data type or made the effort to look it up.